ods_ci/tests/Resources/Page/FeatureStore/FeatureStore.resource

```diff
@@ -6,7 +6,7 @@ Resource         ../../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
 
 
 *** Variables ***
-${FEATURE-STORE-RELEASE-TAG}             adjustment-release-v0.48.1
```

Comment:
Non-blocking comment, but we need to think of pulling this value from somewhere in future.

Reply:
Yes from next release we can make sure the required test changes exists in the release tag itself